### ISSUE TITLE
Compile error with latest HEAD

### ISSUE THREAD

**Issue by bararchy** [client9/libinjection#92]
```
(cd src; make all)
make[1]: Entering directory '/home/unshadow/Desktop/git-projects/libinjection/src'
./sqlparse2c.py < sqlparse_data.json > libinjection_sqli_data.h
  File "./sqlparse2c.py", line 51
    """
      ^
SyntaxError: invalid syntax
Makefile:50: recipe for target 'libinjection_sqli_data.h' failed
make[1]: *** [libinjection_sqli_data.h] Error 1
make[1]: Leaving directory '/home/unshadow/Desktop/git-projects/libinjection/src'
Makefile:5: recipe for target 'all' failed
make: *** [all] Error 2

```

**Comment by bararchy** [comment]
Looking at the issue a little bit more, I can see that becuase I use Arch Linux the default Python version is 3, changing ```#!/usr/bin/env python``` to ```#!/usr/bin/env python2``` raised the following error:

```
(cd src; make all)
make[1]: Entering directory '/home/unshadow/Desktop/git-projects/libinjection/src'
./sqlparse2c.py < sqlparse_data.json > libinjection_sqli_data.h
Traceback (most recent call last):
  File "./sqlparse2c.py", line 131, in <module>
    sys.exit(toc(json.load(sys.stdin)))
  File "/usr/lib/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
Makefile:50: recipe for target 'libinjection_sqli_data.h' failed
make[1]: *** [libinjection_sqli_data.h] Error 1
make[1]: Leaving directory '/home/unshadow/Desktop/git-projects/libinjection/src'
Makefile:5: recipe for target 'all' failed
make: *** [all] Error 2

```

**Comment by zimmerle** [comment]
Safe to be closed as consequence of #144.
